src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```diff
@@ -153,6 +159,14 @@ private void setCommonPreviewParams(SscsCaseData sscsCaseData, String endDate) {
 
     }
 
+    private void commonSetupForInterpreterDescription(SscsCaseData sscsCaseData) {
+        sscsCaseData.setAdjournCaseGenerateNotice("yes");
+        sscsCaseData.setAdjournCaseTypeOfNextHearing("faceToFace");
+        sscsCaseData.setAdjournCaseNextHearingDateType("firstAvailableDate");
+        sscsCaseData.setAdjournCaseTime(AdjournCaseTime.builder().build());
+        sscsCaseData.setAdjournCaseInterpreterRequired("yes");
```

Comment:
I wouldn't do this, set this in setup() then override when necessary in each test when it differs from this.